src/components/draggable/drag-and-drop.stories.mdx

```diff
@@ -0,0 +1,90 @@
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+import { DraggableContainer, DraggableItem } from ".";
+import { Checkbox } from "../../__experimental__/components/checkbox";
+
+<Meta
+  title="Design System/Draggable"
+  parameters={{ info: { disable: true } }}
+/>
+
+# Draggable
+
+Pick, move and replace an order
+
+## Overview 
+
+Draggable is a great component if you need to have interactive components 
+with possibility to change their order with drag and drop mechanics
+
+## Guidance, hints and tips 
+
+Always use `DraggableItem` inside of the `DraggableContainer`
+
+`DraggableItem` doesn't work without `DraggableContainer`
+
+`DraggableItem` require to have an `id` passed to the component
```

Comment:
`require to have an `id` passed to the component` -> `requires an `id` to be passed to the component`